### PR TITLE
Enabled all tests for contributions

### PR DESCRIPTION
**Pull request by jasonrsadler**
Completed enabling tests for contributions.

### CODE REVIEW

File: js/contribution.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ const msecs = {
 class JS extends Contribution {
   constructor () {
     super()
-    this.paymentEnabled = true
-    this.paymentsAllowNonVerified = true
+    this.funding = 'sufficientFunds'
+    this.paymentsContributionAmount = 10
```
Comment (ryanml) [inline js/contribution.js]: Where is this one being used? Any reason we can't just keep it to `PAYMENTS_CONTRIBUTION_AMOUNT` ?

Comment (jasonrsadler) [reply js/contribution.js]: It's unused. Removed.

**Comment by NejcZdovc** [comment]
@jasonrsadler don't forget to change snapshot from synopsis to info

**Comment by jasonrsadler** [comment]
I'd like to address that in a different PR. There are going to be some values that won't be the same across tests (timestamps, votes, etc). I'd like to dig in to snap-shot-it to see if there's a way we can match partials.

**Review by NejcZdovc — APPROVED** [review]
++

**Pull request merged**